src/SFA.DAS.EmployerIncentives.Domain/IncentiveApplications/IncentiveApplication.cs

```diff
@@ -49,8 +49,18 @@ public void Submit(DateTime submittedAt, string submittedByEmail, string submitt
             Model.DateSubmitted = submittedAt;
             Model.SubmittedByEmail = submittedByEmail;
             Model.SubmittedByName = submittedByName;
+
             Model.ApprenticeshipModels.ToList().ForEach(m => m.Phase = IncentivePhase.Create().Identifier);
 
+            // remove ineligible ones
+            Model.ApprenticeshipModels = new List<ApprenticeshipModel>(Model.ApprenticeshipModels.Where(a => a.HasEligibleEmploymentStartDate));
+
+            _apprenticeships.Clear();
+            foreach (var apprenticeshipModel in Model.ApprenticeshipModels.ToList())
+            {
+                _apprenticeships.Add(Apprenticeship.Create(apprenticeshipModel));
+            }
```

Comment:
The comment suggests this could be refactored.

Reply:
Good point.  Now updated.